risedev.yml

```diff
@@ -117,6 +117,7 @@ profile:
       - use: postgres
         port: 8432
         user: postgres
+        password: postgres
```

Comment:
Required, otherwise the PGPASSWORD will be set to `''`, and we encounter connectivity issues for `postgres_query`.